yarn-project/pxe/src/utils/create_pxe_service.ts

```diff
@@ -14,21 +14,40 @@ import { KVPxeDatabase } from '../database/kv_pxe_database.js';
 import { PXEService } from '../pxe_service/pxe_service.js';
 
 /**
- * Create and start an PXEService instance with the given AztecNode.
- * If no keyStore or database is provided, it will use KeyStore and MemoryDB as default values.
- * Returns a Promise that resolves to the started PXEService instance.
+ * Create and start an PXEService instance with the given AztecNode and config.
  *
  * @param aztecNode - The AztecNode instance to be used by the server.
  * @param config - The PXE Service Config to use
  * @param options - (Optional) Optional information for creating an PXEService.
  * @param proofCreator - An optional proof creator to use in place of any other configuration
  * @returns A Promise that resolves to the started PXEService instance.
  */
-export async function createPXEService(
+export function createPXEService(
```

Comment:
Split this function into 2 so that I can pass in the SimulationProviderRecorderWrapper into the PXE in tests.

Reply:
Don't worry too much about these helpers, I'm gonna be refactoring them so we have a nice and convenient way of creating PXEs for server/testing/browser